webapp-django/bin/ci.sh

```diff
@@ -18,12 +18,6 @@ echo "Linting..."
 git ls-files crashstats | grep '\.py$' | xargs flake8 | bin/linting.py
 
 echo "Starting tests..."
-# Override configuration with better defaults for the tests
-export SECRET_KEY="doesn't matter, tests"
-export CACHE_BACKEND="django.core.cache.backends.locmem.LocMemCache"
-export CACHE_LOCATION="crashstats"
-# Change AWS_HOST to empty string so tests use boto.s3.connect_to_region
-export AWS_HOST=
```

Comment:
These are all in `webapp-django/crashstats/settings/test.py` with all the other test-specific settings. Figured it's easier to have one place for test-specific settings.